build/check-images.ts

```diff
@@ -25,7 +25,8 @@ export function checkImageReferences(
   options,
   { url, rawContent }
 ) {
-  const filePaths = new Set();
+  // filePaths is a map of basename to full path
+  const filePaths = new Map<string, string>();
```

Comment:
Maybe we can give the variable a better name:

```suggestion
  // Maps basename to full path (e.g. `example.png` -> `/path/to/content/files/en-us/example/example.png`).
  const imageMap = new Map<string, string>();
```